### DISCUSSION TITLE
Months view for FluentCalendar on Microsoft.Fast.Components.FluentUI

### DISCUSSION THREAD

**Discussion by michaelc06** [microsoft/fluentui-blazor#1632] [General]
Still using .Net 6 and just wondering whether the FluentCalendar will have support for Months view like the .Net 8 version.

**Comment by vnbaaij** [comment]
As .NET 6 only has about 7-8 months of support left, we have not planned on investing time in that.

**Comment by michaelc06** [comment]
Thanks @vnbaaij. I would love to upgrade to .NET 8 but some external libraries we are using don't have support for .NET 8 yet.
So, are you saying there will be no more updates and changes to the .NET 6 version?

**Reply by vnbaaij** [reply]
A .NET 8 application should be able to run with .NET 6 packages without problems, but that is a different discussion 😀

Yes, there will be changes and updates, but we are not porting all of the new(er) functionality back to v3. We only do the ones which are relativly easy (because of limited time available to work on this).
